src/app.scss

```diff
@@ -21,49 +21,33 @@
     font-size: 16px;
 }
 
-// UpdateChoice buttons
 .tabs {
     display: flex;
     justify-content: space-evenly;
     margin-bottom: 50px;
 }
 
-.tabs button {
-    background-color: #ffffff;
-    border-color: #a39797;
-    border-radius: 5px 5px 0 0;
-    padding: 10px 20px;
-}
-:where(.pf-v5-theme-dark) .tabs button {
-    background-color: #212427;
-    border-color: #645d5d;
-    border-radius: 5px 5px 0 0;
-    padding: 10px 20px;
-}
-
-.tabs button.active {
-    border-color: #0077cc;
-    border-width: 3px;
+button{
+    margin-top: 10px;
+    margin-bottom: 10px;
 }
 
-// Regular buttons
-button,
+// Reproduces Patternfly button's style
```

Comment:
How does Patternfly configure their buttons styling? Maybe you can use the same class as them or run some code to make it stylize your button. This would remove the need to maintain this css mirroring.